### PR TITLE
Skip x-pack auditbeat system test as fails on CI

### PR DESCRIPTION
**Pull request by ruflin**
As this is a data test, it should not even be run on CI. This needs further investigation, skipping for now.

### CODE REVIEW

**Comment by ruflin** [comment]
Seems like the system auditbeat test also fails with:
```
Traceback (most recent call last):
  File "c:\python27\lib\unittest\case.py", line 329, in run
    testMethod()
  File "c:\python27\lib\site-packages\nose\loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "c:\python27\lib\site-packages\nose\importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "c:\python27\lib\site-packages\nose\importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "C:\Users\jenkins\workspace\elastic+beats+pull-request+multijob-windows\beat\x-pack\auditbeat\label\windows\src\github.com\elastic\beats\x-pack\auditbeat\tests\system\test_metricsets.py", line 9, in <module>
    from auditbeat_xpack import *
  File "C:\Users\jenkins\workspace\elastic+beats+pull-request+multijob-windows\beat\x-pack\auditbeat\label\windows\src\github.com\elastic\beats\x-pack\auditbeat\tests\system\auditbeat_xpack.py", line 4, in <module>
    from auditbeat import BaseTest as AuditbeatTest
  File "C:\Users\jenkins\workspace\elastic+beats+pull-request+multijob-windows\beat\x-pack\auditbeat\label\windows\src\github.com\elastic\beats\x-pack\auditbeat\tests\system\../../../../auditbeat/tests/system\auditbeat.py", line 11, in <module>
    from metricbeat import BaseTest as MetricbeatTest
ImportError: No module named metricbeat
```

Also going to skip it directly here.

**Comment by ruflin** [comment]
@cwurm Seems like even with this change, it still fails on Windows. Can you take this over?

**Comment by cwurm** [comment]
Opened https://github.com/elastic/beats/pull/9750 so I can iterate on it.

File: x-pack/auditbeat/tests/system/auditbeat_xpack.py [hunk 1]
```diff
@@ -26,6 +27,7 @@ def setUp(self):
         )
 
     # Adapted from metricbeat.py
+    @unittest.skip("Skipped as flaky")
```
[the remark this answers is not in the document]
Comment (cwurm) [reply x-pack/auditbeat/tests/system/auditbeat_xpack.py]: Btw, my comment was wrong. It's not just the `process` metricset failing, it's running the system tests at all on Windows. https://github.com/elastic/beats/issues/9748 has more details.

**Comment by ruflin** [comment]
Closing in favor of https://github.com/elastic/beats/pull/9750

**Pull request closed without merging**